Apps/WebClient/src/ClientApp/test/app.test.ts

```diff
@@ -1,4 +1,6 @@
 import "@/plugins/inversify.config";
+
+import StoreOptionsStub from "@test/stubs/store/storeOptionsStub";
```

Comment:
the path looks redundant
```"@test/stubs/store/options``` sounds better to me

Reply:
I agree but i rather have redundancy than not descriptive enough, in you suggestion the name of the class would be Options

Reply:
The name of the class does not necessarily need to be aligned to the name of the file. However, we can start to do that if we choose to.  At this point I think renaming it to Options would be good as well

Reply:
The name of the class does need to be aligned to the name of the file if it has only one class, we already do that in the C# files